spec/harvesting/harvest_data_setup.rb

```diff
@@ -337,4 +337,6 @@ def cassette_let!(name, &block)
     result
   end
 
+  let!(:invoices) do
+  end
```

Comment:
@woller Why is this necessary?

Reply:
It is not. I added it, when trying to figure out how to get vcr working and forgot to remove it.